llvm/test/CodeGen/AMDGPU/dynamic_stackalloc_wave_reduce.ll

```diff
@@ -0,0 +1,170 @@
+; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx900 -stop-after=amdgpu-isel < %s | FileCheck -check-prefix=GFX9 %s
```

Comment:
This should run to the end and produce final output

Reply:
This test points are similar to `llvm/test/CodeGen/AMDGPU/dynamic_stackalloc.ll`
This test was added to check the selection, more specifically wave reduction intrinsic.